src/alire/alire-builds.adb

```diff
@@ -0,0 +1,135 @@
+with AAA.Strings;
+
+with Alire.Config.Edit;
+with Alire.Directories;
+with Alire.OS_Lib.Subprocess;
+with Alire.Paths.Vault;
+with Alire.Platforms.Current;
+with Alire.Properties.Actions.Executor;
+with Alire.Utils.Tools;
+
+package body Alire.Builds is
+
+   use Directories.Operators; -- "/"
+
+   ----------------------------
+   -- Sandboxed_Dependencies --
+   ----------------------------
+
+   function Sandboxed_Dependencies return Boolean
+   is (not Config.DB.Get (Config.Keys.Dependencies_Shared,
+                          Config.Defaults.Dependencies_Shared));
+
+   -------------------
+   -- To_Msys2_Path --
+   -------------------
+   --  Convert C:\blah\blah into /c/blah/blah. This is needed because otherwise
```

Comment:
Actually rsync is probably overkill since the new folder should either exist or not at all, so a full copy is OK. I was for some reason too focused on using rsync. I take note for a follow-up PR.